modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+
+Default Argo CD instance (Default cluster-scoped instance)::
+A default instance that a {gitops-title} Operator instantiates immediately after its installation, in the `openshift-gitops` namespace, with additional permissions for managing certain cluster-scoped resources.
+
+{gitops-shortname}::
+A declarative way to implement continuous deployment for cloud native applications. In {gitops-shortname}, a Git repository contains deployment resources, which Argo CD then keeps synchronizing with its cluster state.
+
+{gitops-shortname} CLI ({gitops-shortname} `argocd` CLI)::
+A tool to configure and manage {gitops-title} and Argo CD resources from the command line. 
+
+Instance scopes::
+Modes that determine how you want to operate an Argo CD instance. The available modes are cluster-scoped instance and namespace-scoped instance.
+
+Live state::
+The live state of application resources on a target cluster.
+
+Local cluster::
+A cluster where you install Argo CD.
+
+Manifest::
+In the {gitops-shortname} context, a manifest is a YAML representation of Kubernetes resources defined within a {gitops-shortname} repository, with the intent to deploy those resources to a target Kubernetes cluster. Examples include the YAML representation of resources such as `Deployment`, `ConfigMap`, or `Secret`.
+
+Multitenancy::
+A software architecture where a single software instance serves multiple yet distinct user groups.
+
+Namespace-scoped instance (Application delivery instance)::
+A mode in which Argo CD is configured to only manage resources in certain namespaces on a cluster and use them for application delivery.
+
+Notifications Controller (Argo CD Notifications Controller)::
+A controller that continuously monitors Argo CD applications and provides a flexible way to notify users about important changes in the application state. 
+
+Progressive delivery::
+In the {gitops-shortname} context, progressive delivery is a process of releasing application updates in a controlled and gradual manner.
+
+{gitops-title}::
+ An Operator that uses Argo CD as the declarative {gitops-shortname} engine to enable {gitops-shortname} workflows across multicluster OpenShift and Kubernetes infrastructure.
+
+Refresh::
+The process to compare the latest code in the Git repository with the live state and determine the difference. For example, in the Argo CD UI, when you click *Refresh*, Argo CD connects to an application's target Git repository, retrieves the content, and then generates manifests from that content. Argo CD then compares that target state against the live cluster state.
```

Comment:
```suggestion
The process of comparing the latest code in the Git repository with the live state and determining the difference. For example, in the Argo CD UI, when you click *Refresh*, the Argo CD instance connects to an application's target Git repository, retrieves the content, and then generates manifests from that content. the Argo CD instance then compares that target state against the live cluster state.
```

Reply:
Love the 1st part of the suggestion, addressed it. However, I am ignoring the last two suggestions as they change the technical meaning. It is argo CD and not instance.